python-lib/dku_model_fairness_report/fairness_metric_report.py

```diff
@@ -32,29 +38,31 @@ def _compute_confusion_matrix_metrics(y_true, y_pred, advantageous_outcome, samp
         return true_negative, false_negative, true_positive, false_positive
 
     @staticmethod
-    def demographic_parity(y_true, y_pred, advantageous_outcome, sample_weight=None):
+    def demographic_parity(y_true, y_pred, label_list, sample_weight=None):
         """
         demographic_parity just care about y_pred, but we keep y_true to have a homogene api
         """
-        return np.sum(y_pred == advantageous_outcome, dtype=float) / len(y_pred)
+
+        # last label is the advantageous one
+        return np.round(np.sum(y_pred == label_list[-1], dtype=float) / len(y_pred), DkuFairnessConstants.NUMBER_OF_DECIMALS)
```

Comment:
Can you explain why you take label_list[-1] here ?

Reply:
Because we are only interested in the advantageous label, which is the last in the label list